docs/sources/developers/plugins/extend-the-grafana-ui-with-links.md

```diff
@@ -1,46 +1,57 @@
 
-- `grafana/dashboard/panel/menu`: all panel menu dropdowns found in dashboards
+- `grafana/dashboard/panel/menu`: extension point for all panel dropdown menus in dashboards
 
-## Add a link extension within Grafana
+## Add a link extension within a Grafana dashboard panel menu
 
 Here's how you can add a link to the dashboard panel menus in Grafana via your plugin:
 
-Define the link extension in your plugin's `module.ts` file. First, define a new instance of the `AppPlugin` class by using the `configureExtensionLink` method. This method takes an object that describes your link extension, including a `title` property for the link text, a `extensionPointId` that tells Grafana where the link should appear, and a `path` for the user to navigate to your plugin.
+1. Define the link extension in your plugin's `module.ts` file.
+
+1. Define a new instance of the `AppPlugin` class by using the `configureExtensionLink` method. This method requires:
+   - an object that describes your link extension, including a `title` property for the link text
+   - an `extensionPointId` method that tells Grafana where the link should appear
+   - a `path` for the user to go to your plugin
 
 ```typescript
 new AppPlugin().configureExtensionLink({
   title: 'Go to basic app',
-  description: 'Will navigate the user to the basic app',
+  description: 'Will send the user to the basic app',
   extensionPointId: 'grafana/dashboard/panel/menu',
   path: '/a/myorg-basic-app/one', // Must start with "/a/<PLUGIN_ID>/"
 });
 ```
 
-That's it! Your link will be displayed in dashboard panel menus. When the user clicks the link, they will be navigated to the path you defined earlier.
+That's it! Your link will appear in dashboard panel menus. When the user clicks the link, they will be sent to the path that you defined earlier.
 
-_Note: Each plugin is limited to a maximum of two links per extension point._
+> **Note:** Each plugin is limited to a maximum of two links per extension point.
 
 ## Add a link extension using context within Grafana
 
-The above example works for simple cases however you may want to act on information from the panel the user is navigating from. This can be achieved by making use of the `configure` property which takes a function and returns an object that consists of a `title` property for the link text, a `path` to navigate the user to your plugin. Alternatively this function may return `undefined` if there's a need to hide the link for certain scenarios.
+The above example works for simple cases. However, you may want to act on information from the app's panel from which the user is navigating.
+
+To do this, use the `configureExtensionLink` property which takes a function and returns an object that consists of a `title` property for the link text and a `path` to send the user to your plugin.
```

Comment:
This should be `configure` property (it is different from the `configureExtensionLink()` method, it is a property of the object that the `configureExtensionLink()` takes).
```suggestion
To do this, use the `configure` property on the object that is passed to `configureExtensionLink()`. This property takes a function and returns an object that consists of a `title` property for the link text and a `path` to send the user to your plugin.
```